python/health-check/MANIFEST.in

```
@@ -0,0 +1,3 @@
+include src/health_check/config.toml
+recursive-include src/health_check/config *
+recursive-include src/health_check/containers *
```

Missing empty line at the end  of the file
```suggestion
include src/health_check/config.toml
recursive-include src/health_check/config *
recursive-include src/health_check/containers *

```